jaxlib/tools/BUILD.bazel

```diff
@@ -330,22 +331,26 @@ jax_wheel(
     name = "jax_cuda_plugin_wheel",
     enable_cuda = True,
     no_abi = False,
-    # TODO(b/371217563) May use hermetic cuda version here.
-    platform_version = "12",
+    platform_version = cuda_major_version(),
```

Comment:
I think it's better to have `platform_version` passed as constant inside `jax_wheel` macros, and multiple targets for the wheels. So we would have one target for cuda12 jax wheel, another for cuda13 jax wheel etc. This way we won't need extra flags for specifying the CUDA version (we kinda already have too many).

Also, technically we already have major CUDA version saved in `"@cuda_cudart//:version.bzl"`. So we can control the version used for building the wheel with existing `--repo_env=HERMETIC_CUDA_VERSION`, and we won't need to specify it twice in the bazel options.